frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ActionItem.tsx

```diff
@@ -0,0 +1,134 @@
+import { IconButton, Tooltip, styled } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { IAction } from 'interfaces/action';
+import { Fragment } from 'react';
+import GeneralSelect, {} from 'component/common/GeneralSelect/GeneralSelect';
+import { Delete } from '@mui/icons-material';
+import { useProjectEnvironments } from 'hooks/api/getters/useProjectEnvironments/useProjectEnvironments';
+import { useRequiredPathParam } from 'hooks/useRequiredPathParam';
+import mapValues from 'lodash.mapvalues';
+import { useFeatureSearch } from 'hooks/api/getters/useFeatureSearch/useFeatureSearch';
+import {
+    BoxSeparator,
+    Col,
+    InnerBoxHeader,
+    Row,
+    StyledInnerBox,
+} from './InnerContainerBox';
+
+export type UIAction = Omit<IAction, 'id' | 'createdAt' | 'createdByUserId'> & {
+    id: string;
+};
+
+export const Action = ({
+    action,
+    index,
+    stateChanged,
+    onDelete,
+}: {
+    action: UIAction;
+    index: number;
+    stateChanged: (action: UIAction) => void;
+    onDelete: () => void;
+}) => {
+    const { id, action: actionName } = action;
+    const projectId = useRequiredPathParam('projectId');
+    const environments = useProjectEnvironments(projectId);
+    const { features } = useFeatureSearch(
+        mapValues(
+            {
+                project: `IS:${projectId}`,
+            },
+            (value) => (value ? `${value}` : undefined),
+        ),
+        {},
+    );
```

Comment:
I'm curious. How is this different from something like:
```ts
useFeatureSearch({
  project: projectId ? `IS:${projectId}` : undefined
})
```

Reply:
I'm really not sure, I just didn't know how to use the feature and relied on existing code: https://github.com/Unleash/unleash/blob/747b9847177c37026a803b009f8e16fcb449e5a1/frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx#L97-L108